certdb/certdb.go

```diff
@@ -7,32 +7,32 @@ import (
 // CertificateRecord encodes a certificate and its metadata
 // that will be recorded in a database.
 type CertificateRecord struct {
-	Serial    string    `sql:"serial"`
-	CALabel   string    `sql:"ca_label"`
-	Status    string    `sql:"status"`
-	Reason    int       `sql:"reason"`
-	Expiry    time.Time `sql:"expiry"`
-	RevokedAt time.Time `sql:"revoked_at"`
-	PEM       string    `sql:"pem"`
+	Serial    string    `db:"serial"`
+	CALabel   string    `db:"ca_label"`
+	Status    string    `db:"status"`
+	Reason    int       `db:"reason"`
+	Expiry    time.Time `db:"expiry"`
+	RevokedAt time.Time `db:"revoked_at"`
+	PEM       string    `db:"pem"`
 }
 
 // OCSPRecord encodes a OCSP response body and its metadata
 // that will be recorded in a database.
 type OCSPRecord struct {
-	Serial string    `sql:"serial"`
-	Body   string    `sql:"body"`
-	Expiry time.Time `sql:"expiry"`
+	Serial string    `db:"serial"`
+	Body   string    `db:"body"`
+	Expiry time.Time `db:"expiry"`
 }
 
 // Accessor abstracts the CRUD of certdb objects from a DB.
 type Accessor interface {
 	InsertCertificate(cr *CertificateRecord) error
 	GetCertificate(serial string) (*CertificateRecord, error)
-	GetUnexpiredCertificates() ([]*CertificateRecord, error)
+	GetUnexpiredCertificates() ([]CertificateRecord, error)
```

Comment:
I want to keep some consistencies between get one record vs get a slice of record, but it could be hard. Is it possible we don't change this function signature?